content/examples/clone_and_copy.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "clone and copy"
+date: 2023-06-22
+draft: false
+---
+
+Because of Cairo's ownership rules, it's necessary to specify how data can be duplicated. For this the `Clone` and `Copy` traits are used. `Clone` specifies how a type instance can be *cloned*, meaning its data duplicated and assigned a new owner. The behaviour specified by `Clone` can be **arbitrarily complicated**. Instead `Copy` requires `Clone` and it specifies that a type can be *bitwise-copied*, meaning that its cloning mechanism is simple and known, instead of arbitrary.
```

Comment:
"Type instance" sounds a bit strange to me. I would say something like "an instance of a given type" maybe?